tests/test_network_extensions.py

```diff
@@ -453,9 +453,12 @@ def test_geo_data():
 
     pd.testing.assert_frame_equal(n.get_extensions('linePosition'), line_expected)
 
-
```

Comment:
A test can be added here to try to get/add/remove a 'cgmesMetadataModels' extension on a network.
The creating dataframe can be as follow :
```python
pd.DataFrame.from_records(index='id', columns=['id', 'subset', 'description', 'version', 'modelingAuthoritySet', 'profiles', 'dependentOn', 'supersedes'], data=[('sshId', 'STEADY_STATE_HYPOTHESIS', 'SSH description', 1, 'http://powsybl.org', 'steady-state-hypothesis', 'ssh-dependency1/ssh-dependency2', '')])

```